hbw/app/models/hbw/activiti/yml_api.rb

```diff
@@ -5,7 +5,11 @@ class YMLAPI
 
       class << self
         def build
-          new(YAML.load_file(Rails.root.join('hbw/config/yml_api.default.yml')))
+          if Rails.env.test?
```

Comment:
Adapter should not depend on environment. Pass file as a parameter.

Reply:
Done.